DuckDuckGo/OnboardingExperiment/DaxDialogs/DaxDialogBrowsersComparisonView.swift

```diff
@@ -47,13 +48,18 @@ struct DaxDialogBrowsersComparisonView: View {
                     .font(Font.system(size: 20, weight: .bold))
 
 
-                    Group {
+                    VStack(spacing: 24) {
                         BrowsersComparisonChart(privacyFeatures: BrowsersComparisonModel.privacyFeatures)
 
-                        Button(action: action) {
-                            Text(UserText.DaxOnboardingExperiment.BrowsersComparison.cta)
-                        }
-                        .buttonStyle(PrimaryButtonStyle())
+                        OnboardingActions(
+                            viewModel: .init(
```

Comment:
Do we even need to have this model if it just contain strings?

Reply:
Good question! This view was implemented and used in old onboarding and it uses a model…to avoid rewriting it I’m using this one. Once we remove the old onboarding we can avoid having a model and pass the titles.